students/BrianNisonger/Lesson3/assignment/basic_operations.py

```diff
@@ -0,0 +1,66 @@
+from peewee import *
+from customer_model import Customer
+from customer_model import database
+
+
+def model_to_dict(model):
+    model_dict = {
+        'customer_id': model.customer_id,
+        'name': model.first_name,
+        'last_name': model.last_name,
+        'home_address': model.home_address,
+        'phone_number': model.phone_number,
+        'email_address': model.email_address,
+        'status': model.status,
+        'credit_limit': model.credit_limit
+    }
+    return model_dict
```

Comment:
Good idea. I did notice in the peewee documentation, you can return select statements as dictionaries, but does seem silly to do a select for the one entry you want.